docs/index.rst

```diff
@@ -48,41 +48,50 @@
 
     * **Intent-based programmability**: Inmanta optimizes service development and maintenance for telecom operators and service providers through its unifying, model-driven methodology for intent-based orchestration.
 
-    * Inmanta's powerful domain-specific language (DSL) simplifies service creation and management, and is based on infrastructure as code (IaC) principles to provide a unified way to automate multi-domain and multi-vendor services. The embedded DSL enables the development of modular building blocks that make abstraction of low-level details, enabling re-usability across use cases.
-    * Inmanta's intent-based programmability provides out-of-the-box self-healing, safe roll-back, detailed dry run and seamless service upgrades for enhanced stability and resilience.
+      * Inmanta's powerful domain-specific language (DSL) simplifies service creation and management, and is based on infrastructure as code (IaC) principles to provide a unified way to automate multi-domain and multi-vendor services. The embedded DSL enables the development of modular building blocks that make abstraction of low-level details, enabling re-usability across use cases.
+      * Inmanta's intent-based programmability provides out-of-the-box self-healing, safe roll-back, detailed dry run and seamless service upgrades for enhanced stability and resilience.
 
     * **Vendor agnostic**: Inmanta Service Orchestrator is truly open and vendor agnostic for all network layers, domains and OSS/BSS. Service providers can integrate with 3rd party solutions as well as a wide range of open-source technologies to build a best-in-class, all-encompassing solution.
 
-    * Interoperability through pluggable adapters and open APIs
-    * API-ification of orchestrated services to easily plug services into the OSS/BSS environment
-    * Support for brownfield environments by fine-grained roll-out
+      * Interoperability through pluggable adapters and open APIs
+      * API-ification of orchestrated services to easily plug services into the OSS/BSS environment
+      * Support for brownfield environments by fine-grained roll-out
 
     The Inmanta Service Orchestrator product is based on mature technology backed by 15+ years of research and interaction with companies offering telecom and cloud services.
 
 
 .. toctree::
-    :maxdepth: 1
+    :maxdepth: 3
 
     quickstart
     install
     architecture
     language
+    moduleguides
     model_developers
     platform_developers
     lsm/index
     administrators
+    extensions
     faq
     glossary
     reference/index
     troubleshooting
     changelogs
 
+Additional resources
+--------------------
+
+* `Inmanta User Mailinglist <https://groups.google.com/forum/#!forum/inmanta-users>`_
+* `Inmanta Developer Mailinglist <https://groups.google.com/forum/#!forum/inmanta-devel>`_
+* `Inmanta Twitter <https://twitter.com/inmanta_com>`_
+
```

Comment:
This section I really don't know about. I think it's good to add it back, but again I'd ask Bart when he's back.

Reply:
@bartv do you have an opinion on this?

Reply:
These mailing lists are not in use anymore so I would just remove the entire section.